test/helpers/increaseTime.js

```diff
@@ -0,0 +1,54 @@
+var latestTime = require('./latestTime');
+
+// Increases testrpc time by the passed duration in seconds
+
+function increaseTime(duration) {
```

Comment:
this and other functions here should make it clear in their names that they modify testrpc time. There is concept of time in javascript already and by default that's the time one talks about when talking about time in javascript

Reply:
:+1: